src/AuthApi.js

```diff
@@ -10,17 +14,47 @@ export type UserCredentials = {
 
 class AuthApi {
   static login = ({ password, userName }: UserCredentials): Promise<Object> =>
-    fetchJSON(`${config.HOST}token/`, {
+    fetchJSON(`${CONFIG.HOST}token/`, {
       body: `grant_type=password&userName=${userName}&password=${password}`,
       headers: { 'Content-type': 'application/x-www-form-urlencoded' },
       method: 'POST',
     });
 
-  // todo waiting for server implementation
-  static register = (): Promise<Object> => Promise.resolve();
+  static changePassword = (
+    changePasswordFields: ChangePasswordFormFields,
+  ): Promise<Object> =>
+    // eslint-disable-next-line
+    fetch(`${CONFIG.HOST}api/account/change-password`, {
```

Comment:
I'd recommend putting a `/` at the end of these.. Sometimes my server gets finicky without them.